### PR TITLE
toggleable manual ability popup destruction

### PR DESCRIPTION
**Pull request by ghoulslash**
## Description
Option to keep ability popup around and manually remove

To keep around, add this in a battle script before `call BattleScript_AbilityPopUp`:
- `setbyte sFIXED_ABILITY_POPUP, TRUE`

Then, when you want to remove it, call `destroyabilitypopup`

fixed-popup

### CODE REVIEW

File: src/battle_interface.c [hunk 1]
```diff
@@ -3110,11 +3113,19 @@ static void SpriteCb_AbilityPopUp(struct Sprite *sprite)
...
+            if (!gBattleScripting.fixedPopup)
+                sprite->tFrames--;
         }
     }
 }
 
+void DestroyAbilityPopUp(u8 battlerId)
+{
+	gSprites[gBattleStruct->abilityPopUpSpriteIds[battlerId][0]].tFrames = 0;
+	gSprites[gBattleStruct->abilityPopUpSpriteIds[battlerId][1]].tFrames = 0;
```
Comment (LOuroboros) [inline src/battle_interface.c:3125]: Should use empty spaces in these lines instead of tabs :eyes:

File: src/battle_interface.c [hunk 2]
```diff
@@ -3062,6 +3062,9 @@ void CreateAbilityPopUp(u8 battlerId, u32 ability, bool32 isDoubleBattle)
         gSprites[spriteId1].tRightToLeft = FALSE;
         gSprites[spriteId2].tRightToLeft = FALSE;
     }
+    
+	gBattleStruct->abilityPopUpSpriteIds[gBattleAnimAttacker][0] = spriteId1;
+	gBattleStruct->abilityPopUpSpriteIds[gBattleAnimAttacker][1] = spriteId2;
```
Comment (LOuroboros) [inline src/battle_interface.c:3067]: Should use empty spaces in these lines instead of tabs :eyes:

**Comment by ghoulslash** [comment]
fixed formatting. ready for re review

**Review by ultima-soul — CHANGES_REQUESTED** [review]
Functions well.

**Review by ultima-soul — APPROVED** [review]

**Review by AsparagusEduardo — APPROVED** [review]

**Pull request merged**